src/dash/student_view/Profile.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { updateUser } from '../../redux/actions.js';
+import { updateUser } from '../../redux/actions/user_actions.js';
 
 class Profile extends React.Component {
 
```

Comment:
in the handleEdit method you should try to combine the setState calls. The way it's currently being executed you will be making two setState calls every time, you should be able to combine those into one.